var/spack/repos/builtin/packages/blitz/package.py

```diff
@@ -31,3 +31,10 @@ class Blitz(AutotoolsPackage):
     url = "https://github.com/blitzpp/blitz/tarball/1.0.0"
 
     version('1.0.0', '9f040b9827fe22228a892603671a77af')
+
+    def build_targets(self):
+        return ['lib']
```

Comment:
Not really, @mamelara was looking for a way to add make targets, I mentioned overriding `build` and that it would be trivial to add a `make_targets` member, @alalazo liked the idea, so I figured I'd go for it. I don't think it introduces problems, unless someone incorrectly overrides both `build_targets` and `build`, which would be silly.